hasktorch-skeleton.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version:       2.1
+cabal-version:       1.24
```

Comment:
This is the same `cabal-version` used by hasktorch. When I upgrade to GHC 8.10.4 without changing this, I get

```
hasktorch-skeleton.cabal:0:0: error:
Unsupported cabal-version 2.1. See https://github.com/haskell/cabal/issues/4899.
```